os/logm/logm_tashcmds.c

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/logm.h>
+#include <string.h>
+#include <stdlib.h>
+#include <apps/shell/tash.h>
+#include <stdio.h>
+#include "logm.h"
+
+const static tash_cmdlist_t logm_tashmds[] = {
```

Comment:
Logm is a kernel context, but registering a command at TASH is a user context. You can not call this at protected build. What is your plan for this?